### PR TITLE
Add netstandard2.0 target for ServerTelemetryChannel and remove Newtonsoft.Json dependency (#800)

### PR DESCRIPTION
**Pull request by clguimanMSFT**
Add netstandard2.0 target for ServerTelemetryChannel and remove Newtonsoft.Json dependency (#800)

Fix Issue #800  .
Replaced JavaScriptSerializer with DataContractjsonSerializer (exists in netstandard 2.0 & net45)
I had to update the BackendResponse member attributes with IsRequired=true so BackoffLogicManagerTest.IfContentIsUnexpectedJsonNullIsReturned won't fail.

- [x] I ran [Unit Tests](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/.github/CONTRIBUTING.md) locally.

For significant contributions please make sure you have completed the following items:

- [ ] Design discussion issue #
- [ ] Changes in public surface reviewed
- [ ] CHANGELOG.md updated with one line description of the fix, and a link to the original issue.
- [ ] The PR will trigger build, unit test, and functional tests automatically. If your PR was submitted from fork - mention one of committers to initiate the build for you.

### CODE REVIEW

**Comment by cijothomas** [comment]
Please add a test project targetting 2.0 mimicking this https://github.com/Microsoft/ApplicationInsights-dotnet/tree/develop/Test/ServerTelemetryChannel.Test/NetCore.Tests

**Comment by clguimanMSFT** [comment]
@cijothomas I've added the 2.0 tests

**Comment by MS-TimothyMothra** [comment]
@clguimanMSFT Can you please update the changelog?

**Comment by clguimanMSFT** [comment]
@MS-TimothyMothra should I add the changes under 2.7.0-beta1?

**Comment by MS-TimothyMothra** [comment]
@clguimanMSFT you can call it 2.6.2.
we're going to get this out right away.

File: Common.props [hunk 1]
```diff
@@ -21,7 +21,7 @@
     <CodeAnalysisRuleSet>$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildProjectDirectory), 'ApplicationInsightsSDKRules.ruleset'))\ApplicationInsightsSDKRules.ruleset</CodeAnalysisRuleSet>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+  <PropertyGroup Condition="'$(Configuration)' == 'Release' and '$(TargetFramework)'!='netstandard2.0'">
```
[the remark this answers is not in the document]
Comment (pharring) [reply Common.props]: The condition should be on the "RunCodeAnalysis" property. The other properties (particularly Optimize and DefineConstants) should still be set.

[the remark this answers is not in the document]
Comment (clguimanMSFT) [reply Common.props]: Fixed it

**Pull request merged**